src/bin/tikv-ctl.rs

```diff
@@ -1999,6 +2012,24 @@ fn read_fail_file(path: &str) -> Vec<(String, String)> {
     list
 }
 
+// check if the key word "ldb" exists as the first argv and format the cmd line
+// to meet the requirements of ldb_tool.
+fn check_run_ldb_cmd() -> Option<Vec<String>> {
+    let mut res = Vec::new();
+    for x in env::args_os() {
```

Comment:
Should we skip `tikv-ctl` and `ldb`?

Reply:
what do you mean?

Reply:
`env::args_os` should return `["tikv-ctl" "ldb", ...]`?

Reply:
Yes, `check_run_ldb_cmd` will remove `"ldb"` and return like `["tikv-ctl", "cmd1", ...]`, which is required by rocksdb.